samples/dnn/trackers/siamrpnpp.py

```diff
@@ -44,55 +48,29 @@ def track(self, x):
         cls, loc = self.rpn_head.forward(outNames)
         return {'cls': cls, 'loc': loc}
 
-class Anchors:
-    """ This class generate anchors.
-    """
-    def __init__(self, stride, ratios, scales, image_center=0, size=0):
-        self.stride = stride
-        self.ratios = ratios
-        self.scales = scales
-        self.image_center = image_center
-        self.size = size
-        self.anchor_num = len(self.scales) * len(self.ratios)
-        self.anchors = self.generate_anchors()
-
-    def generate_anchors(self):
-        """
-        generate anchors based on predefined configuration
-        """
-        anchors = np.zeros((self.anchor_num, 4), dtype=np.float32)
-        size = self.stride**2
-        count = 0
-        for r in self.ratios:
-            ws = int(np.sqrt(size * 1. / r))
-            hs = int(ws * r)
-
-            for s in self.scales:
-                w = ws * s
-                h = hs * s
-                anchors[count][:] = [-w * 0.5, -h * 0.5, w * 0.5, h * 0.5][:]
-                count += 1
-        return anchors
 
 class SiamRPNTracker:
     def __init__(self, model):
         super(SiamRPNTracker, self).__init__()
+        self.windowing = "cosine"
+        self.base_size = 8
+        self.exemplar_size = 127
+        self.instance_size = 255
         self.anchor_stride = 8
         self.anchor_ratios = [0.33, 0.5, 1, 2, 3]
         self.anchor_scales = [8]
-        self.track_base_size = 8
-        self.track_context_amount = 0.5
-        self.track_exemplar_size = 127
-        self.track_instance_size = 255
```

Comment:
Why these variables should be renamed?

Reply:
Is it normal to have two similar classes to have different names for the same attributes?